### DISCUSSION TITLE
Idea: Interoperability with Solid

### DISCUSSION THREAD

**Discussion by grue-bleen** [blockprotocol/blockprotocol#409] [Ideas]
The [solid project](https://solidproject.org) provides the perfect standards based base for the blockprotocol to live and flourish upon.  The architecture enables a semantic web that restores user control on user data, and inherits a standards first approach thanks to [Sir Tim Berners-Lee](https://solidproject.org/team) and undoubtedly many other W3C contributors.  The solidproject is in sore need of a great linked data content authoring solution, and the blockprotocol would be a natural fit.

I plead ignorance on both projects but I hope smarter minds find ways to make this happen so that we can all benefit from the ensuing semantic goodness.

**Comment by ghost** [comment]
Hi cyrilchemparathy.

> The [solid project](https://solidproject.org/) provides the perfect standards based base for the blockprotocol to live and flourish upon. The architecture enables a semantic web that restores user control on user data, and inherits a standards first approach thanks to [Sir Tim Berners-Lee](https://solidproject.org/team) and undoubtedly many other W3C contributors. The solidproject is in sore need of a great linked data content authoring solution, and the blockprotocol would be a natural fit.
> I plead ignorance on both projects but I hope smarter minds find ways to make this happen so that we can all benefit from the ensuing semantic goodness.

great idea - So, assuming you have a public repository where any visual component lives, that would be fantastic to have a public audit of how certain: websites, blogs, web apps work.

*I could be wrong and I hope someone corrects me, but for me your point of view is interesting. On the one hand we have a network protocol to store certain data, on the other hand we have a protocol to say how certain visual components are made and distributed.*

But... but recently I spoke with the development team of block protocol, and the block protocol does not involve or say that it should use a certain database like orbit-db/mysql etc, just as I imagine that there would not be a protocol used like solid, remote-storage, wedav, ftp - all network protocols can be used, but should not or cannot be involved with block protocol.


*I recently asked if there was a need for the block protocol to use webmention, to notify anyone that a certain component is used or has been changed. Please see this: [(question) Block protocol and Webmention.io?](https://github.com/blockprotocol/blockprotocol/discussions/833) - perhaps this idea will be shelved or not.*

**Reply by grue-bleen** [reply]
[dokie.li](https://dokie.li) and [plume](https://thewebalyst.solidcommunity.net/plume/) are probably the closest to what you're looking for as a starting point.

**Reply by ghost** [reply]
Hi grue-bleen, great feedback.
